installer/msi/build-msi.ps1

```diff
@@ -6,40 +6,113 @@
     .
 .PARAMETER Arch
     The architecture, x86 or x64.
+.PARAMETER Version
+    The version of the driver.
+.PARAMETER VersionLabel
+    The version label of the driver.
 #>
 Param(
-  [string]$Arch,
-  [string]$Version,
-  [string]$VersionLabel,
-  [string]$UpgradeCode
+    [string]$Arch,
+    [string]$Version,
+    [string]$VersionLabel
+    # [string]$UpgradeCode
```

Comment:
There's a lot of formatting changes in this file (xml-tools). This is open to debate on whether we want to pass in an `$UpgradeCode`.